.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Don't request an update of a package,
+        [We have a script for that](https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt).
+        However, a quality pull request may help.
+  - id: xuname
+    type: input
+    attributes:
+      label: System Info
+      description: Output of `xuname` (part of [`xtools`](https://man.voidlinux.org/xtools.1))
+      placeholder: Void 5.x.y_z x86_64-musl ...
+    validations:
+      required: true
+  - id: packages
+    type: input
+    attributes:
+      label: Package(s) Affected
+      description: Affected package(s) including version
+      placeholder: foo-1.0.2_5, bar-5.6.7_1, baz-0.0.3_5, ...
```

Comment:
Could we mention that people can get these strings with `xbps-query -p pkgver foo`?

Reply:
addressed